### ISSUE TITLE
More diverse user colors

### ISSUE THREAD

**Issue by silverwind** [StylishThemes/IRCCloud-Dark#6]
We could do 27 colors based on

``` html
<a data-name="silv3rwind">
```

With selectors like

``` css
.nick-colors [data-name^="a"] { /* first color */ }
.nick-colors [data-name^="A"] { /* first color */ }
.nick-colors [data-name^="a"] { /* second color */ }
.nick-colors [data-name^="A"] { /* second color */ }
```

People without alphabetically beginning names should probably fallback to a 28th color.

**Comment by Mottie** [comment]
Actually, it looks like [most browsers now support css filters](http://caniuse.com/#feat=css-filters), so instead of defining new colors, why not mess with brightness, contrast and/or hue-rotate of the existing nick colors?

**Comment by silverwind** [comment]
Yeah, that'd actually be easier to read in the style. We should do a base color and rotate hue from there (28 steps).

**Comment by Mottie** [comment]
Nice idea!

On a non-related topic, why does the readme tag still show v0.1.1?

**Comment by silverwind** [comment]
Good question. I manually tagged that release. Let me try again with `npm version patch`.

**Comment by Mottie** [comment]
I wonder if [this is related](https://github.com/badges/shields/issues/161#issuecomment-41812007)... I think that is saying there needs to be more requests for the tag before it'll update.

**Comment by silverwind** [comment]
Possibly, but it's working for me now at 1.0.1.

**Comment by Mottie** [comment]
Maybe it needs the "v" in front? But cool :P

**Comment by silverwind** [comment]
Yeah, another possibilty. Anyways, I'll not push this to userstyles yet, as I'm not sure I'll keep the stuff in the header hidden like it is now.
